.github/workflows/scripts/setup-dependencies.sh

```diff
@@ -55,29 +55,24 @@ function mod_install() {
   cat /proc/spl/kstat/zfs/chksum_bench
   echo "::endgroup::"
 
-  echo "::group::Reclaim and report disk space"
-  # remove 4GiB of images
-  sudo systemd-run docker system prune --force --all --volumes
+  echo "::group::Optimize storage for ZFS testings"
+  # remove swap and umount fast storage
+  # 89GiB -> rootfs + bootfs with ~80MB/s -> don't care
+  # 64GiB -> /mnt with 420MB/s -> new testing ssd
+  sudo swapoff -a
 
-  # remove unused software
-  sudo systemd-run --wait rm -rf \
-    "$AGENT_TOOLSDIRECTORY" \
-    /opt/* \
-    /usr/local/* \
-    /usr/share/az* \
-    /usr/share/dotnet \
-    /usr/share/gradle* \
-    /usr/share/miniconda \
-    /usr/share/swift \
-    /var/lib/gems \
-    /var/lib/mysql \
-    /var/lib/snapd
-
-  # trim the cleaned space
-  sudo fstrim /
+  # this one is fast and mounted @ /mnt
+  # -> we reformat with ext4 + move it to /var/tmp
+  DEV="/dev/disk/azure/resource-part1"
+  sudo umount /mnt
+  sudo mkfs.ext4 -F $DEV
```

Comment:
If we already format and remount ext4 - maybe:
- mount with `noatime,barrier=0` ?
- disable journal?
- other speedups I've missed?

Reply:
> mount with noatime,barrier=0 ?
> disable journal?

These two are done .... thanks for pointing out.

> maybe compilation will be faster too from this ssd? If it's so - we may move code onto it

Compiling is done once on startup in ~10..20 minutes. This should be okay.